schema.json

```diff
@@ -0,0 +1,107 @@
+                        "default": "",
+                        "examples": [],
+                        "pattern": "[easy|medium|hard]"
+                    },
+                    "description": {
+                        "$id": "#/items/anyOf/0/properties/description",
+                        "type": "string",
+                        "title": "The description schema",
+                        "description": "An explanation about the purpose of this instance.",
+                        "default": "",
+                        "examples": []
+                    },
+                    "reference": {
+                        "$id": "#/items/anyOf/0/properties/reference",
+                        "type": "string",
+                        "title": "The reference schema",
+                        "description": "An explanation about the purpose of this instance.",
+                        "default": "",
+                        "examples": []
+                    },
+                    "image": {
+                        "$id": "#/items/anyOf/0/properties/image",
+                        "type": "string",
+                        "title": "The image schema",
+                        "description": "An explanation about the purpose of this instance.",
+                        "default": "",
+                        "examples": [],
+                        "format": "uri"
+                    },
+                    "dateAdded": {
+                        "$id": "#/items/anyOf/0/properties/dateAdded",
+                        "type": "string",
+                        "title": "The dateAdded schema",
+                        "description": "An explanation about the purpose of this instance.",
+                        "default": "",
+                        "examples": [],
+                        "format": "date-time"
+                    },
+                    "tags": {
+                        "$id": "#/items/anyOf/0/properties/tags",
+                        "type": "array",
+                        "title": "The tags schema",
+                        "description": "An explanation about the purpose of this instance.",
+                        "default": [],
+                        "examples": [],
+                        "additionalItems": true,
+                        "items": {
+                            "$id": "#/items/anyOf/0/properties/tags/items",
+                            "anyOf": [
+                                {
+                                    "$id": "#/items/anyOf/0/properties/tags/items/anyOf/0",
+                                    "type": "string",
+                                    "title": "The first anyOf schema",
+                                    "description": "An explanation about the purpose of this instance.",
+                                    "default": "",
+                                    "examples": []
```

Comment:
I'm wondering if we should not set a fixed list of allowed tags in there. @vikaspotluri123 wdyt?

Reply:
It's not a fixed list, is it?